### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by sQVe** [zimfw/zimfw#447] [General]
Hey :wave:,

I just made the dive and **updated** my fork of `zimfw` to the **latest version and I was amazed**. The new design and implementation allows me to archive my fork fully and just keep the dotfiles. 

This framework is indispensable to me and my daily work so I just wanted to take the opportunity and give a **huge thank you to everyone involved**. A special thank you to @ericbn for driving version 1.0.0 forward.

Feel free to close this whenever :wink: 

:heart:

**Comment by ericbn** [comment]
Hey @sQVe, that's great feedback! Thank you for the kind words!

**Comment by Arcitec** [comment]
@ericbn I wanted to thank you too, but didn't want to create a new ticket just for that, so I found this one.

Hello! :)

Thank you so freaking much for creating ZIM. I love everything about it. It's lean and clean, super fast and beautiful. I love how well organized and minimalistic the code is. I was always terrified of the bloated mess that is Oh My Zsh. And Prezto is cleaner but still a mess. Zim is exactly what I was searching for!

It's... perfect!

Wrote a thread to guide people towards Zim here:

https://www.reddit.com/r/linuxadmin/comments/rhg7wx/zsh_frameworks/

// Johnny

**Reply by ericbn** [reply]
@Bananaman, thanks a lot for spreading the love for Zim!

**Comment by Konfekt** [comment]
Just joining in at Christmastime, thank you @ericbn for crafting this great ZSH framework
